### DISCUSSION TITLE
Unable to change codeblock language

### DISCUSSION THREAD

**Discussion by danfate** [outline/outline#4599] [Self Hosting] [answered]
Everything works fine except i can't change codeblock language in Macos chrome. It always "javascript" after I click the dropdown menu and select other language. However, it works in iphone chrome.

No error or exception in dev tools console. Also tried in incognito window and safari.

Any one knows why?

v0.66.3 & 0.67.0-pre.1

Edit:
Having some test on Windows system, language change works too. still don't know why it doesn't work on mac.

**Comment by tommoor** [comment, marked as answer]
This has been reported in the past but I've never been able to reproduce it to work on a fix, it's unclear in what situations it occurs

**Reply by danfate** [reply]
I just download firefox for mac, and it works too. Only chrome and safari not work. 
So i find that there's an extension that i installed both on chrome and safari ,and also enabled in incognito mode --- Enpass.
It a password manager. After disable the extension, now i can change the language.
Still not show why it will break the function. 
Maybe this information can help you reproduce ?
